rep-0003.rst

```diff
@@ -180,6 +179,46 @@ Buildsystem support:
 
   - build from source
 
+Kinetic Kame (May 2016 - May 2021)
+----------------------------------
+Required Support for:
+
+- Ubuntu Wily (15.10)
+- Ubuntu X (16.04)
+
+Minimum Requirements:
+
+- C++03
+- C++11
+
+- Python 2.7
+
+  - Python 3.4 not required, but testing against it is recommended
+
+- Lisp SBCL 1.2.4
+- CMake 3.0.2
+- Boost 1.58
+
+Exact or Series Requirements:
+
+- Ogre3D 1.9.x
+- Gazebo 5 (TODO(esteve): or Gazebo 6?)
```

Comment:
If gazebo is available upstream we should use the upstream version. That's part of going upstream, you get that specific version. If we overlay a custom version we break compatibility with anyone else using the upstream package.